javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,68 @@
+//age calculator
+const yearOfBirth=2000;
+let yearFuture= 2040;
+let age=yearFuture-yearOfBirth;
+console.log("You will be "+ age+ " years old in "+yearFuture);
+
+//dog age calculator
+const dogYearOfBirth=2017 ;
+let dogYearFuture=2027;
+let dogYear=dogYearFuture-dogYearOfBirth;
+let shouldShowResultInDogYears=true;
+if(shouldShowResultInDogYears){
+     dogYear=dogYear*7;
+     console.log("Your dog will be " +dogYear +" dog years old in "+ 2027);
+}
+else{
+    dogYear
+    console.log("Your dog will be "+dogYear+" human years old in "+dogYearFuture);
+}
+
+//house estimator 
+
+const peterHouse = {
+    width: 8,
+    depth: 10,
+    high: 10,
+    gardenSizeInM2:100,
+    cost:2500000
+  };
+
+  const juliaHouse = {
```

Comment:
peter house and juliahouse have the same keys 'width, depth....'. for.ex if you have company employee data of 1000 employees, you have to repeat this code 1000 times. Not for this but find how can we write in a more optimize way.